CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Ewon Flexy Extensions Library Changelog
```

Comment:
Typically the title for release commits would be in the following format:

`RELEASE: v1.13.11`

I just checked quickly and the title *shouldn't* affect any of our scripts/automation though.